src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
```

Comment:
fix indents for better radability
```suggestion
  shape: 'triangle';
  color: 'red' | 'green' | 'blue';
  private a: number;
  private b: number;
  private c: number;
```

Reply:
It's a linter requirement, so it won't even pass the github tests.